content/en/docs/contributing/e2e.md

```diff
@@ -5,91 +5,97 @@ weight: 70
 type: "docs"
 ---
 
-cert-manager has an end-to-end test suite that verifies functionality against a
+cert-manager has an extensive end-to-end (e2e) test suite that verifies functionality against a
 real Kubernetes cluster.
 
-This test takes around 30 minutes, it will be run on every PR in our cluster.
+The full end-to-end test suite can take a long time to complete and is run against every pull
+request made to the cert-manager project.
 
-> Note: you can see the status of each commit on the master branch at
-> [`testgrid.k8s.io`](https://testgrid.k8s.io/jetstack-cert-manager-master).
-> You can join the Google group
-> [`cert-manager-dev-alerts`](https://groups.google.com/g/cert-manager-dev-alerts)
-> in order to receive a notification by email whenever a commit on master
-> fails.
+Unless you've made huge changes to the cert-manager codebase - or to the end-to-end
+tests themselves - you probably don't _need_ to run the tests locally. If you do want to
+run the tests, though, this document explains how.
```

Comment:
nit: I assumed by `-` you meant em-dashes as per https://practicaltypography.com/trademark-and-copyright-symbols.html? Three dashes `---` are rendered as an em-dash in Hugo:

```suggestion
Unless you've made huge changes to the cert-manager codebase---or to the end-to-end
tests themselves---you probably don't need to run the tests locally. If you do want to
run the tests, though, this document explains how.
```

Before:

![image](https://user-images.githubusercontent.com/2195781/126307539-0f363304-dcb2-4145-85fd-f0e0adb29df3.png)


After:

![image](https://user-images.githubusercontent.com/2195781/126307486-980e8132-b6c0-41ca-9f8e-98c822cdf9ae.png)

Reply:
Did you mean to link to that page specifically? I'm not sure how it relates to dashes

I'm very indifferent to dash styling; I'd be totally content seeing and using `-` on any page or in any piece of text. I'll switch to emdashes though just to be fancy :wink: